modules/dxp/apps/commerce/commerce-shop-by-diagram-web/src/main/java/com/liferay/commerce/shop/by/diagram/web/internal/helper/CSDiagramCPTypeHelperImpl.java

```diff
@@ -3,18 +3,19 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-package com.liferay.commerce.shop.by.diagram.web.internal.util;
+package com.liferay.commerce.shop.by.diagram.web.internal.helper;
 
 import com.liferay.account.model.AccountEntry;
 import com.liferay.commerce.media.CommerceMediaProvider;
 import com.liferay.commerce.product.model.CPDefinition;
 import com.liferay.commerce.product.permission.CommerceProductViewPermission;
 import com.liferay.commerce.product.service.CPDefinitionLocalService;
+import com.liferay.commerce.shop.by.diagram.helper.CSDiagramCPTypeHelper;
 import com.liferay.commerce.shop.by.diagram.model.CSDiagramSetting;
 import com.liferay.commerce.shop.by.diagram.service.CSDiagramSettingLocalService;
 import com.liferay.commerce.shop.by.diagram.type.CSDiagramType;
 import com.liferay.commerce.shop.by.diagram.type.CSDiagramTypeRegistry;
-import com.liferay.commerce.shop.by.diagram.util.CSDiagramCPTypeHelper;
+import com.liferay.commerce.shop.by.diagram.web.internal.util.CSDiagramSettingUtil;
```

Comment:
why are we importing this now?

Reply:
because before it was in the same package `com.liferay.commerce.shop.by.diagram.web.internal.util` and therefore it was not imported now by changing the package to `com.liferay.commerce.shop.by.diagram.web.internal.helper` it needs to be imported